io_scene_nif/modules/nif_import/collision/havok.py

```diff
@@ -306,7 +306,8 @@ def import_bhkpackednitristrips_shape(self, bhk_shape):
                                   bhk_tri.v_3 - vertex_offset))
                 else:
                     continue
-
+            # TODO: I was scared to change this to an f-string. I'm not sure
+            # TODO: if that's applicable here.
```

Comment:
You should be able to change it to `f"poly{subshape_num:d}"` as it is just the name of the mesh that will be generated.

Reply:
Updated.